docs/gallery/general/plot_configurator.py

```diff
@@ -0,0 +1,113 @@
+"""
+How to Configure Term Validations
+=================================
+
+This is a user guide for how to curate and take advantage of configuration files in
+order to more easily validate terms within datasets or attributes.
+
+Introduction
+-------------
+Users will create a configuration YAML file that outlines the fields (within a neurodata type)
+they want to be validated against a set of allowed terms. 
+After creating the configuration file, users will need to load the
+configuration file with the :py:func:`~pynwb.load_type_config` method.
+With the configuration loaded, every instance of the neurodata
+types defined in the configuration file will have the respective fields wrapped with a
+:py:class:`~hdmf.term_set.TermSetWrapper`.
+This automatic wrapping is what provides the term validation for the field value.
+For greater control on which datasets and attributes are validated
+against which sets of allowed terms, use the 
+:py:class:`~hdmf.term_set.TermSetWrapper` on individual datasets and attributes instead.
+You can follow the 
+`TermSet tutorial in the HDMF documentation 
+<https://hdmf.readthedocs.io/en/stable/tutorials/plot_term_set.html#sphx-glr-tutorials-plot-term-set-py>`_
+for more information.
+
+To unload a configuration, simply call :py:func:`~pynwb.unload_type_config`.
+We also provide a helper method to see the configuration that has been loaded:
+:py:func:`~pynwb.get_loaded_type_config`
+
+
+How to make a Configuration File
+--------------------------------
+To see an example of a configuration file, please refer to
+`<https://github.com/NeurodataWithoutBorders/pynwb/tree/dev/src/pynwb/config/nwb_config.yaml>`_.
```

Comment:
```suggestion
`<https://github.com/NeurodataWithoutBorders/pynwb/tree/dev/docs/gallery/general/nwb_gallery_config.yaml>`_.
```

What do you think about pointing to this file and removing the `src/pynwb/config` folder? I think the gallery config files serve the same purpose as `src/pynwb/config` unless I am missing something.

Reply:
So the purposes of both are slightly different. The one in gallery is for the tutorial, but the one in `src` is the NWB curated one that we would recommend and maintain over time.

Reply:
I see. It wasn't clear to me and I think it would not be clear to the user that we are recommending and maintaining that configuration file for users to use. I know you want to get this merged, so let's move forward, and I'll add this suggestion to the docs refactor issue.